libobs/obs-output.c

```diff
@@ -1151,12 +1278,17 @@ get_audio_conversion(struct obs_output *output)
 	return output->audio_conversion_set ? &output->audio_conversion : NULL;
 }
 
-static size_t get_track_index(const struct obs_output *output,
-			      struct encoder_packet *pkt)
+static size_t get_encoder_index(struct obs_output *output,
+				struct encoder_packet *pkt)
 {
+	// Mutiple audio tracks are more common
```

Comment:
Is it possible to use `pkt->type` to not have to iterate multiple times depending on packet type?